src/nlp/counters.jl

```diff
@@ -57,7 +57,11 @@ end
 Increment counter `s` of problem `nlp`.
 """
 @inline function increment!(nlp::AbstractNLPModel, s::Symbol)
-  setproperty!(nlp.counters, s, getproperty(nlp.counters, s) + 1)
+  NLPModels.increment!(nlp, Val(s))
```

Comment:
Thanks @AntoninKns for the PR. Since we are in the NLPModels.jl package, is there a reason why we need `NLPModels.increment!` everywhere?

Reply:
I don't know I copied some code with that in it so I thought it was necessary but I tested it without and it worked.